paper-input-behavior.html

```diff
@@ -180,6 +180,13 @@
       return this.$.input;
     },
 
+    /**
+     * Validates the input element and sets an error style if needed.
+     */
+     validate:function () {
+       return this.querySelector('paper-input-container').validate(this.inputElement, true);
```

Comment:
Or fire an event from `iron-validatable-behavior` that the container listens for to set `invalid`.